homeassistant/components/wemo/fan.py

```diff
@@ -304,6 +304,9 @@ def turn_on(self, speed: str = None, **kwargs) -> None:
         else:
             self.set_speed(speed)
 
+        self._update()
```

Comment:
Why do we call `self._update` here? It will be called at the end of the service call again since this entity is polling.

Reply:
I didn't realize that HASS will call the update method immediately following the turn_on call. I can remove it.